alf/examples/sac_fetchpush.gin

```diff
@@ -0,0 +1,2 @@
+include 'sac_fetchreach.gin'
```

Comment:
I don't think it's necessary to check in lots of gin files for all the fetch environments? Typically an example gin file should correspond to figures and gif files.